libtransact/src/workload/runner.rs

```diff
@@ -521,12 +532,142 @@ fn submit_batch(
         })
 }
 
+fn check_batch_status(
+    mut status_links: ExpectedBatchResults,
+    auth: &str,
+) -> Result<ExpectedBatchResults, WorkloadRunnerError> {
+    for (status_link, (target, expected_result)) in status_links.clone() {
+        // remove the part of the target URL that is in the beginning of the status link
+        let mut parts = target.splitn(2, "/scabbard");
```

Comment:
libtransact shouldn't know about scabbard specifically.

Reply:
Created a method that will remove overlapping parts of the target/status link instead of specifically using the "/scabbard" substring